# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../adm8211/linux-2.6.19.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- adm8211/adm8211_hw.c.orig	2005-03-23 12:12:02.000000000 +0100
+++ adm8211/adm8211_hw.c	2007-01-02 16:54:08.000000000 +0100
@@ -2468,9 +2468,13 @@
 	dev->set_mac_address = adm8211_set_mac_address;
 	dev->hard_start_xmit = adm8211_hard_start_xmit;
 
+#if WIRELESS_EXT >= 12
+#if WIRELESS_EXT < 20
 	dev->get_wireless_stats = adm8211_get_wireless_stats;
+#endif
 	dev->wireless_handlers =
 		(struct iw_handler_def *) &adm8211_iw_handler_def;
+#endif
 
 	init_timer(&priv->timer);
 	priv->timer.data = (unsigned long) dev;
